opentasks/build.gradle

```diff
@@ -49,5 +49,6 @@ dependencies {
     compile project(':opentasks-provider')
     compile 'com.google.android.apps.dashclock:dashclock-api:2.0.0'
     compile 'com.github.dmfs:color-picker:1.0'
+    compile 'com.github.dmfs.jems:iterators:1.7.1'
```

Comment:
Just use `org.dmfs:iterators:1.7.1`. There is no need to specify the github jems version. In fact, the maven dependency name will remain the same even with the jems version. Same goes for `optional`.

Reply:
Ok, updated.